src/GregClient/Utility/Utilities.cs

```diff
@@ -12,37 +12,72 @@ namespace Greg.Utility
 {
     public static class AppSettingMgr
     {
+        private static XmlDocument debugDoc;
 
+        static AppSettingMgr()
+        {
+            string configPath = $"{typeof(AppSettingMgr).Assembly.Location}.config";
+
+            if (!File.Exists(configPath)) return;
+
+            try
+            {
+                debugDoc = new XmlDocument();
+                debugDoc.Load(configPath);
```

Comment:
Try to load the debug config file only once. (I see no use cases for having to always try and read the file on each debug item request)